packages/cli/src/commands/deploy/helper.ts

```diff
@@ -37,135 +26,42 @@ type UpsertFieldPluginFunc = (args: {
   scope: Scope
 }) => Promise<{ id: number }>
 
-export const deploy: DeployFunc = async ({
-  skipPrompts,
-  token,
+type DecidePackageName = (params: {
+  name?: string
+  skipPrompts: boolean
+  dir: string
+}) => Promise<{ error: false; name: string } | { error: true }>
+export const getPackageName: DecidePackageName = async ({
```

Comment:
oh if I read this diff correctly, are we merging `getPackageName` and `decidePackageName`? If so, I see the inconsistency to fix here:

```
export const getPackageName: DecidePackageName
```

Reply:
true, going to fix it. I just wanted to make it more clear what the function does